app/styles/app.scss

```diff
@@ -142,11 +166,14 @@ h2 {
 }
 
 .main_title {
-  background-color: $blue;
-  color:$white;
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets